oneshot.d/browser-update-default-data

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+MOZEMBED_DIR=/home/$USER/.mozilla/mozembed
+BROWSER_DATA_DIR=/usr/share/sailfish-browser/data
+
+if [ ! -d "$MOZEMBED_DIR" ]; then
+    mkdir -p $MOZEMBED_DIR
+fi
+
+if [ ! -f "$MOZEMBED_DIR/ua-update.json" ]; then
+    while IFS='' read -r line || [[ -n "$line" ]]; do
+        # Skip comment lines
+        if [[ $line !=  *\/\/* ]] ; then
+            echo "$line" >> "$MOZEMBED_DIR/ua-update.json"
+        fi
+    done < "$BROWSER_DATA_DIR/ua-update.json.in"
```

Comment:
I'd suspect comparing with pattern is bash specific feature. Also don't think this allows comments at end of lines.

If above functionality is enough, wouldn't it be just simpler with
 grep -v // $BROWSER_DATA_DIR/ua-update.json.in > $MOZEMBED_DIR/ua-update.json